app/controllers/evaluation_controller.rb

```diff
@@ -61,12 +61,19 @@ def export
 
   def edit
     @evaluation = Evaluation.find(evaluation_id)
+    @instructors = Instructor.select_menu_options
     render layout: "layouts/centered_form"
   end
 
   def update
+    key_attrs, other_attrs = split_attributes(evaluation_params)
     @evaluation = Evaluation.find(evaluation_id)
-    @evaluation, _ = Evaluation.create_if_needed_and_update(@evaluation.key, evaluation_params)
+
+#    @evaluation = Evaluation.where(key_attrs).first_or_initialize
+    @evaluation.assign_attributes(other_attrs)
+    @evaluation.save
+
+#    @evaluation, _ = Evaluation.create_if_needed_and_update(@evaluation.key, evaluation_params)
```

Comment:
This broke my code.
No matter what I choose form the drop down on the edit page :instructor is set to "" evaluation_params(instructor)=>""